gulpfile.js/tasks/stylus.js

```diff
@@ -6,30 +6,30 @@ module.exports = function (config) {
     const autoprefixer = require('autoprefixer-stylus')
     const changed = require('gulp-changed')
     const gulpIf = require('gulp-if')
-    const handleErrors = require('../utils/handleErrors')
+    // const handleErrors = require('../utils/handleErrors')
```

Comment:
remove?

Reply:
will be removed before merge in master. still doing some testing